daffodil-lib/src/main/scala/org/apache/daffodil/util/Coroutines.scala

```diff
@@ -73,8 +71,12 @@
     * Call when a co-routine resumes another (to provide a result of some sort)
     * and then terminates. The coroutine calling this must return from the run()
     * method immediately after calling this.
+    *
```

Comment:
Highlight: minor tweak to Coroutines to allow different types of data flowing on each path between two coroutines.